src/App.js

```diff
@@ -1,13 +1,30 @@
 import "./App.css";
+import { Link, Route, Switch } from "react-router-dom";
+import Home from "./Home";
+import Movies from "./Movies";
+import People from "./People";
+import Locations from "./Locations";
+import "./App.css";
 
 function App() {
   return (
     <div className="app">
-      <main>
-        <h1>Hello, world!</h1>
-      </main>
+      <nav>
+        <Link to="/">
+          <img src="https://images.pexels.com/photos/2801980/pexels-photo-2801980.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260" alt="logo" />
+        </Link>
+        <Link to="/movies">Movies</Link>
```

Comment:
You should use NavLinks inside your NavBar so you can have the activated property